src/test/translate.ts

```diff
@@ -0,0 +1,51 @@
+import * as fs from 'node:fs';
+
+type TranslationNode = {
```

Comment:
strange types, looks like they are the same as LocaleTranslation, Translations

Reply:
this function no longer exists